controllers/enable_authorino.go

```diff
@@ -103,16 +105,7 @@ func (r *OpenshiftServiceMeshReconciler) createAuthConfig(namespace *v1.Namespac
 
 	authConfig.Labels[keyValue[0]] = keyValue[1]
 	authConfig.Spec.Hosts = authHosts
-
-	// Reflects oauth-proxy SAR settings
-	authConfig.Spec.Authorization[0].KubernetesAuthz.ResourceAttributes = &authorino.Authorization_KubernetesAuthz_ResourceAttributes{ //nolint:nosnakecase //reason external library
-		Namespace: authorino.StaticOrDynamicValue{Value: namespace.Name},
-		Group:     authorino.StaticOrDynamicValue{Value: "kubeflow.org"},
-		Resource:  authorino.StaticOrDynamicValue{Value: "notebooks"},
-		Verb:      authorino.StaticOrDynamicValue{Value: "get"},
-	}
-
-	authConfig.Spec.Identity[0].KubernetesAuth.Audiences = getAuthAudience()
+	authConfig.Spec.Identity[0].KubernetesAuth.Audiences = []string{namespace.Name + "-api"}
```

Comment:
How would it work for e.g. project with only workbench enabled? At this point it expects the user to be able to "get" notebooks in this namespace. 

It's a similar approach in other components IIRC. Do we foresee some granularity of the roles? Can you help me understand how is this supposed to replace it?

Reply:
This is all Host based.

Workbenchs + Notebooks are bound to the "*.opendatahub.openshift.ai" domain which is the openatahub level authconfig with oauth/single sign on configured. Which namespace they belong to doesn't matter.

Models are bound to a various of different domains, aka "*.namespace.srv.cluster.local", "*.namespace.apps.openshift.ai". These are configured in the project level authconfig.

In the current impl the ProjectLevel authconfig would still exist in the Project namespace if only workbenches are used, but the workbench is reached on different URL pattern so it won't be active.